mlflow/R/mlflow/tests/testthat/test-model-h2o.R

```diff
@@ -7,6 +7,15 @@ predictors <- setdiff(colnames(iris), prediction)
 train <- iris[idx[1:100], ]
 test <- iris[idx[101:nrow(iris)], ]
 
+# Installing most recent h2o package, see https://docs.h2o.ai/h2o/latest-stable/h2o-docs/downloading.html#install-in-r
+if ("package:h2o" %in% search()) { detach("package:h2o", unload=TRUE) }
+if ("h2o" %in% rownames(installed.packages())) { remove.packages("h2o") }
+pkgs <- c("RCurl","jsonlite")
+for (pkg in pkgs) {
+  if (! (pkg %in% rownames(installed.packages()))) { install.packages(pkg) }
+}
+install.packages("h2o", type="source", repos=(c("http://h2o-release.s3.amazonaws.com/h2o/latest_stable_R")))
```

Comment:
Do we know this link to be stable/reliable?

Reply:
I believe it's more reliable than CRAN, since CRAN might have update delays compared to the source or pypi (as seen when their CRAN download page was taken down for a few days when they updated their package).

Reply:
Cool, we could probably just revisit this if this link causes any trouble then.